.travis.yml

```diff
@@ -189,4 +190,4 @@ jobs:
 # Syntax and more info: https://docs.travis-ci.com/user/notifications
 notifications:
   email:
-    - janusgraph-ci@googlegroups.com
\ No newline at end of file
+    - janusgraph-ci@googlegroups.com
```

Comment:
Why do removed the last line?

Reply:
I didn't remove anything. IntelijIDEA adds new line symbol by default on the end of each file. I.e. the line was not removed. IntelijIDEA just added a hidden character. If necessary I can remove it but I think this is OK

Reply:
Okay

Reply:
I'm guessing what happened here is that this file was modified previously by someone with a different setting for newline at EOF, and we'll have these spurious diffs every time someone with a different configuration setting updates a file.

If you'd like consistent editor settings across everyone's environments, please update `https://github.com/JanusGraph/janusgraph/blob/master/.editorconfig` and set up your editor accordingly, e.g., there's a [plugin for IntellijIDEA](https://github.com/JetBrains/intellij-community/tree/master/plugins/editorconfig) which supports this config file, and it's supported for a lot of other editors as well: https://editorconfig.org/#download

Reply:
@mbrukman Thank you. I configured my IDEA to not change those files. I think this won't happen again.